### ISSUE TITLE
Forbidden exceptions on unique constraint violations?

### ISSUE THREAD

**Issue by AlexKolpa** [inventid/live-image-resize#122]
I was curious why iaas returns 403 forbidden on a unique constraint violation. In general those exceptions would be treated as a 409 conflict, right?

**Comment by rogierslag** [comment]
Could also be a 409 indeed. In this case I opted for a 403 since the token request is denied

**Comment by AlexKolpa** [comment]
Yes, denied , but not forbidden. Forbidden suggests I'm missing the proper rights to make this requests.

**Comment by rogierslag** [comment]
I wouldn't mind, but note this would require a major release as is may break backwards compatibility for clients
